docs/configuring.md

```diff
@@ -380,6 +380,64 @@ Also don't ask questions to the user; just use default values
 
     It makes no sense to define this in `copier.yml`.
 
+### `jinja_extensions`
+
+-   Format: `List[str]`
+-   CLI flags: N/A
+-   Default value: []
+
+Additional Jinja2 extensions to load in the Jinja2 environment. Extensions can add
+filters, global variables and functions, or tags to the environment.
+
+Examples of extensions you can use:
+
+-   [`jinja_markdown.MarkdownExtension`](https://github.com/jpsca/jinja-markdown):
+    provides a `markdown` tag that will render Markdown to HTML using
+    [PyMdown extensions](https://facelessuser.github.io/pymdown-extensions/).
+-   [`jinja2_slug.SlugExtension`](https://pypi.org/project/jinja2-slug/#files): provides
+    a `slug` filter using [unicode-slugify](https://github.com/mozilla/unicode-slugify).
+-   [`jinja2_time.TimeExtension`](https://github.com/hackebrot/jinja2-time): adds a
+    `now` tag that provides convenient access to the
+    [arrow.now()](http://crsmithdev.com/arrow/#arrow.factory.ArrowFactory.now) API.
```

Comment:
I think it would be very helpful to list here cookiecutter extensions, because those won't appear while searching pypi, and would help people coming from cookiecutter to preserve some features they might be using.

Reply:
Totally. But maybe we could show how to do the same with Ansible filters, without a dependency to cookiecutter, since it's possible.